res/openvpn-gui-res-no.rc

```diff
@@ -272,9 +272,9 @@ BEGIN
     IDS_NFO_STATE_FAILED_RECONN "Status: Kunne ikke koble til på nytt"
     IDS_NFO_RECONN_FAILED "Tilkobling til %s feilet."
     IDS_NFO_STATE_SUSPENDED "Status: Hvilemodus"
-    IDS_ERR_READ_STDOUT_PIPE "Feil under lesning fra OpenVPN StdOut pipe."
-    IDS_ERR_CREATE_EDIT_LOGWINDOW "Opprettelse av RichEdit LogWindow feilet!"
-    IDS_ERR_SET_SIZE "Set Size feilet!"
+    IDS_ERR_READ_STDOUT_PIPE "Feil under lesning fra OpenVPN standard ut-kanal."
+    IDS_ERR_CREATE_EDIT_LOGWINDOW "Opprettelse av statusloggvindu feilet!"
+    IDS_ERR_SET_SIZE "'Set Size' feilet!"
```

Comment:
The ERR_SET_SIZE is about font size and font face for the status log messages. "Setting font size failed"?